tests/test_packages.py

```diff
@@ -0,0 +1,111 @@
+import textwrap
+
+from lektor import packages
+
+
+def create_plugin(package_dir, plugin_name, setup):
+    plugin_dir = package_dir / plugin_name
+    plugin_dir.mkdir(parents=True)
+    setup_py = plugin_dir / "setup.py"
+    setup_py.write_text(textwrap.dedent(setup))
+    return plugin_dir
+
+
+def test_install_local_package_without_dependency(tmp_path):
+    packages_dir = tmp_path / "packages"
+    plugin_dir = create_plugin(
+        packages_dir,
+        "dependency",
+        setup="""
+        from setuptools import setup
+
+        setup(
+            name='dependency',
+        )
+        """,
+    )
+
+    install_dir = tmp_path / "target"
+    install_dir.mkdir()
+
+    packages.install_local_package(install_dir.as_posix(), plugin_dir.as_posix())
```

Comment:
We're on Python 3.6 now, so passing these as Paths should work

Reply:
The original code works with string paths - so I didn't want to change that so as to not create a difference in how the code runs under tests vs. in real life. I can change it if it is important to you - but I wouldn't advise it.